### ISSUE TITLE
Feature: web terminal controller

### ISSUE THREAD

**Issue by fanux** [labring/sealos#1335]
https://github.com/labring/fist/blob/master/terminal/README.md

https://book.kubebuilder.io/quick-start.html

```
apiVersion: terminal.sealos.io/v1
kind: Terminal
metadata:
  name: aws-infra-demo
spec:
  user: xxx
  token: xxx
  keepalived: 5min
```

Then `kubectl apply -f terminal.yaml` will create a web terminal.

**Comment by SignorMercurio** [comment]
Isn't this issue closed by #1398?

**Comment by gitccl** [comment]
> Isn't this issue closed by #1398?

This feature is not yet complete, I will continue to submit a PR  tomorrow.
